corehq/motech/openmrs/tasks.py

```diff
@@ -192,51 +191,54 @@ def import_patients_to_domain(domain_name, force=False):
     for importer in get_openmrs_importers_by_domain(domain_name):
         if not _should_import_today(importer) and not force:
             continue
+        import_patients_with_importer.delay(importer.to_json())
```

Comment:
how big can this JSON be? Just concerned about big arguments in celery<->rabbit.

Reply:
I was wondering the same thing. An `OpenmrsImporter` is used to create cases, and its size is proportional to the number of case properties we want to set when the case is created. Usually that is small (so, maybe, yagni), but technically the hard limit is the number of columns in a MySQL table, which is 4096.

I'm going to merge this so that the partner gets the benefit of it asap, and I'll follow up in a second PR to pass an ID instead.